### ISSUE TITLE
`ignoreDeps` in group presets causes error when combined with `matchUpdateTypes`

### ISSUE THREAD

**Issue by RahulGautamSingh** [renovatebot/renovate#31986]
### Describe the proposed change(s).

We added `ignoreDeps` to the group presets in [this PR](https://github.com/renovatebot/renovate/pull/9836) as a hack to shorten the onboarding PR description. 

But, now we cannot combine the group presets with `matchUpdateTypes` as the preset contains `ignoreDeps`. So, while the hack fixed one problem it introduced another.

We need to find another way to shorten the onboarding PR description

**Comment by rarkins** [comment]
Are you sure you have the link right? I don't see mention of `ignoreDeps` in https://github.com/renovatebot/renovate/pull/9836/files

**Comment by RahulGautamSingh** [comment]
My bad, updated the link

**Comment by rarkins** [comment]
Are you sure that PR added ignoreDeps anywhere? Looks like it was already there and was just commented

**Comment by RahulGautamSingh** [comment]
True. I did some mistake while checking blame and could not find the `ignoreDeps` before that PR.

It seems the `ignoreDeps` has always been there, since that file was created. I don't think it is needed though.

**Comment by rarkins** [comment]
I've marked this as low priority - a community member can investigate if it's possible to remove it
